docs: context/summary, verifing bundles, using hw wallet, AGORIC_NET in agops-cli

0xpatrickdev opened the pull request:
- see #13

0xpatrickdev committed:
docs: context/summary, verifing bundles, using hw wallet, AGORIC_NET in agops-cli

0xpatrickdev committed:
fix(install-bundle): ensure bundle path is prefaced with @ symbol for correct parsing

dckc commented:
While we're tweaking the README, note the 4 commands to change the price can be reduced to:

```
oracle setPrice --pair stATOM.USD --keys $WALLET,$WALLET_2 --price 11.99
```

The 4 commands (accept x2, wallet send x2) to accept invitations are still necessary prerequisites.

dckc commented on README.md:
```diff
@@ -23,6 +31,16 @@ agd keys show [dev-local] -a
+
 ### 3. Request faucet funds + Provision Wallet
 
 Setup a smart-wallet and request funds here: https://devnet.faucet.agoric.net/, using the address from the previous step.
```
Do we note how much IST is needed?

I think with the exception of the 1 bundle with the lower-case fix, all these bundles are in [mainnet1B-bundle-ids.md](https://gist.github.com/dckc/4088fb3cf7b568ce81ddabab2817f336#file-mainnet1b-bundle-ids-md), btw.

dckc reviewed (approved):
very nice!

I have a couple suggestions, but we can always improve the docs. I'm fine landing this PR as-is.

0xpatrickdev committed:
docs: update bundle deploy instructions

dckc commented:
Can I squeeze one more in, since this hasn't landed?

How about a link to:

 - [Aug 23 Office Hours: stATOM testing, ... · agoric-sdk #8240](https://github.com/Agoric/agoric-sdk/discussions/8240)

and maybe some eye candy...

![image](https://github.com/0xpatrickdev/agoric-vault-collateral-proposal/assets/150986/6e8ffd82-affe-457b-85e0-32ae1eb6ad40)


![image](https://github.com/0xpatrickdev/agoric-vault-collateral-proposal/assets/150986/93c0f7f2-060c-4ba8-9efe-97d2fe94de96)

0xpatrickdev committed:
docs: updates

- gov and dapp screenshots
- how to add other collaterals
- link to office hours demo
- note about existing bundles (no need to redeploy)

0xpatrickdev commented:
Updated this PR once more. Think it's good to go if you want to give a quick once-over @dckc . 


I am missing the note about cost. I can add that in a follow up PR. I am still not certain on the number, I'm not sure which one of these (if any) is correct:
<img width="712" alt="image" src="https://github.com/0xpatrickdev/agoric-vault-collateral-proposal/assets/11021913/0d641244-3aae-4c73-ae36-a4e6ee7e3d00">

dckc commented:
## Storage: 0.002 IST/byte

Column E looks right. 2,900IST for 1450kb.

Using `agd --node=https://main.rpc.agoric.net:443 query swingset params` we get:

```
beans_per_unit:
- beans: "1000000000000"
  key: feeUnit
- beans: "2000000000"
  key: storageByte
fee_unit_price:
- amount: "1000000"
  denom: uist
```

In spreadsheet form:

key | note | beans | bytes | IST
-- | -- | -- | -- | --
feeUnit |   | 1000000000000|   |  
storageByte |   | 2000000000| 1 | 0.002
  | per Kb |   | 1000| 2.00
  | b1-8fb... |   | 1,456,129| 2,912.258|

This is also consistent with [an Apr 24 comment of mine](https://github.com/Agoric/agoric-sdk/issues/7366#issuecomment-1521115022).

dckc commented:
reminder:
> I think with the exception of the 1 bundle with the lower-case fix, all these bundles are in [mainnet1B-bundle-ids.md](https://gist.github.com/dckc/4088fb3cf7b568ce81ddabab2817f336#file-mainnet1b-bundle-ids-md),

So $10k is less relevant.

dckc commented:
p.p.s. it's probably still worth a "How much does it cost to upload contracts?" discussion.
